test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -876,76 +877,123 @@ void ProtoConverter::visit(Statement const& _x)
 	}
 }
 
-void ProtoConverter::visit(Block const& _x)
+void ProtoConverter::openScope(vector<string>&& _x)
```

Comment:
If you never move from `_x`, then using `&&` does not help. You can just use `const&`.